src/lib/metadataAPI.js

```diff
@@ -37,7 +36,7 @@ const metadataAPI = {
      * @param {Object} [options.sharedLinkPassword] - shared link password if any
      * @return {Promise} XHR promise
      */
-    getMetadata(id, scope, template, { apiHost, token, sharedLink, sharedLinkPassword }) {
+    getMetadata(id, scope, template, { api, apiHost, token, sharedLink, sharedLinkPassword }) {
```

Comment:
Should this module extend `API` somehow?

Reply:
I don't think extending will help in this instance, as we still wont have access to the Preview API instance so won't have access to the loaded interceptors.

Reply:
Thanks for the tip on how it is done in the Node SDK makes sense once I saw the example.